utils/get_codes_from_unicode_emoji_data_files.py

```diff
@@ -164,6 +164,12 @@ def extract_names(xml, lang):
             if lang == "de":
                 emoji_name = emoji_name.replace("\u201c", "").replace("\u201d", "")
 
+            if lang == "fa":
+                emoji_name = emoji_name.replace('\u200c',"_")
+                emoji_name = re.sub("_+","_",emoji_name)
+                emoji_name = re.sub(" +"," ",emoji_name)
```

Comment:
This line is unnecessary, spaces are already removed at this point

Reply:
got it!